demos/coreMQTT/mqtt_demo_basic_tls.c

```diff
@@ -84,7 +84,7 @@
  * @brief The root CA certificate belonging to the broker.
  */
 #ifndef democonfigROOT_CA_PEM
-    #define democonfigROOT_CA_PEM    tlsATS1_ROOT_CERTIFICATE_PEM
```

Comment:
Maybe should be a #error

Reply:
Have to use `#define` instead of `#error` until the "PR Build checks" can be updated to define this macro at build time. Once, the CI job is updated to define this macro, I will change it to a `#error` for better user experience about the significance of the macro.

Reply:
Just take care because #error is not part of the C standard, it may not work on all compilers